docs/QuickstartBuildAgent.mdx

```diff
@@ -0,0 +1,154 @@
+
+Click on the <div className="button darkblue_button">Create</div> button to create the agent.  This will initiate the build of an Agent, which will take roughly 20 minutes to complete, and can be monitored by clicking on the <div className="button darkblue_button"><BellOutlined /></div> in the header bar to view the Notifications page.
+
+:::info
+
+Subsequent iterations to the user-defined code within the agent will not require agent rebuild, and so will be much faster.  A typical update will take less than 10 seconds to complete, and will not require agent re-installation.
+
+:::
+
+A completed agent build will look like this on the Notifications page:
+
+<div class="text--center">
+<img alt="Agent build notification" src="https://ganymede-bio.mo.cloudinary.net/agent/AgentBuildNotificationsPage_20240125.png"/>
+</div>
+
+### Step 3: Download and install the agent
+
+If you're not already on the local PC where the agent will be installed, navigate to that PC and sign into your tenant (as described in [step 1](#step-1-sign-into-ganymede)).
+
+#### Step 3a: Sync Windows time (if necessary)
+
+Right click on the time in the lower right hand corner of the Windows taskbar and select "Adjust date/time".  In the window that opens, click on the "Sync now" button to ensure that the time is synchronized with the internet time server.
+
+#### Step 3b: Create a folder to watch
+
+Create a new folder on your Desktop by right-clicking on the Desktop and selecting "New" -> "Folder".  Name the folder "watch_folder".
+
+:::note
+
+If you choose a pre-existing folder, be aware that the agent will, by default, not process any files that exist in the folder prior to launch.
+
+:::
+
+#### Step 3c: Download the agent
+
+When the agent build is complete, the agent will be available for download from the Connections pane.
+
+<div class="text--center">
+<img alt="Agent download" src="https://ganymede-bio.mo.cloudinary.net/agent/AgentDownload_20240125.png"/>
+</div>
+
+
+On the local PC where the agent will be installed, sign into your tenant (as described in [step 1](#step-1-sign-into-ganymede)) and download the installer for the Windows agent by hovering over the <div className="button dark_gray_button"><DownloadOutlined /> Download</div> button and selecting Windows.
+
+Once downloaded, run the executable to start the installation process.  Name the connection "Quickstart Agent", and type in the path to the folder created in [step 3b](#step-3b-create-a-folder-to-watch) in the "Watched folder" field.  
+
+:::warning
+
+Take care when configuring the watch folder path, since a configuration error here may be challenging to debug. Specifically:
+* Ensure that forward slashes are used in the path specification.
```

Comment:
I don't think this is true anymore? I've not followed this for awhile at least

Reply:
oh neat - does single backslash work?

Reply:
Yeah. I just copy and paste the path from the windows explorer